src/libcaf.h

```diff
@@ -313,4 +314,10 @@ void PREFIX (unlock) (caf_token_t, size_t, int, int *, char *, int);
 void PREFIX (event_post) (caf_token_t, size_t, int, int *, char *, int);
 void PREFIX (event_wait) (caf_token_t, size_t, int, int *, char *, int);
 void PREFIX (event_query) (caf_token_t, size_t, int, int *, int *);
+
+/* Language extension */
+#ifdef HAVE_MPI
+MPI_Fint PREFIX (get_communicator) (caf_team_t *);
```

Comment:
I haven't understood what this is for. But it seems to be the only reason for mpi.h to be needed. When this an non-standard Fortran extension it should be provided by a second include-file. This allows users to selectively build against libcaf.h where only that is needed and to libcaf-ext.h where to extended api is needed. 

Furthermore, is this function as a language extension incomplete, because it lacks documentation describing its purpose, necessity, possible errors and caveats.